common/duplicated_node_checker/include/duplicated_node_checker/duplicated_node_checker_core.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Tier IV, Inc.
```

Comment:
nits:
```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
I saw other packages like system_error_monitor use "Tier IV" in the license. Did something changed along the way?